src/stackdriver-nozzle/README.md

```diff
@@ -1,19 +1,86 @@
+## stackdriver-nozzle
 
+### Installation
 
-## Stackdriver nozzle
+```sh
+go get github.com/cloudfoundry-community/gcp-tools-release/src/stackdriver-nozzle
+```
+
+### Configuration
+
+`stackdriver-nozzle` is configured through the following environment variables:
+
+#### Firehose
+
+- `FIREHOSE_ENDPOINT` - the CF API endpoint; e.g., `https://api.bosh-lite.com'
+- `FIREHOSE_EVENTS` - comma-separated list of events to pass to Stackdriver;
+  valid events are `LogMessage`, `ValueMetric`, `CounterEvent`, `Error`,
+  `ContainerMetric`, `HttpStart`, `HttpStop`, and `HttpStartStop`
+- `FIREHOSE_USERNAME` - CF username; defaults to `admin`
```

Comment:
should we put some info on permissions required by this user, here or in its own section?